app/models/permission.rb

```diff
@@ -0,0 +1,31 @@
+class Permission
+  def self.all
+    {
```

Comment:
This could be a constant, called from the .all method:

```
PERMISSIONS = { ... }.freeze

def self.all
  PERMISSIONS
end
```
 Allows for easier refactoring out of config block - in case it needs to be moved into a YAML file for instance.

Reply:
done